client/src/app/tasks/tasks.component.spec.ts

```diff
@@ -56,13 +51,9 @@ describe('TasksComponent', () => {
         { id: 1, title: 'test title', description: 'description', due_date: null }
       ], status: 200
     };
-    const action = TasksAction.taskFetchSuccess({ tasks });
-    store.dispatch(action);
 
+    taskFacadeSpy.getList.and.returnValue(of(tasks));
```

Comment:
__[TSLint]__ of is deprecated: use {@link scheduled} instead `scheduled([a, b, c], scheduler)` (deprecation) ([link](https://sider.review/gh/repos/72339652/pulls/68#issue-4037366))

<sub>posted by Sider</sub>